docs/docker.md

```diff
@@ -0,0 +1,142 @@
+
+The following example illustrates a scenario where you want to set the PDA_PDNS_API_KEY app setting in a secure fashion. You have the secret API key stored at /run/secrets/PDA_PDNS_API_KEY.
+
+    PDA_PDNS_API_KEY_FILE=/run/secrets/PDA_PDNS_API_KEY
+
+This would be the equivalent of running:
+
+    export PDA_PDNS_API_KEY=CONTENTS_OF_SECRET_FILE
+
+#### Approach #2
+
+With this approach, you may create traditional PowerDNS Admin config files and map them to an accessible location inside the container. Then you can pass an environment variable to indicate where to load the file from.
+
+For example, say you have mapped a python configuration file to /srv/app/configs/my-config.py in the container. You would add the following environment variable to your deployment configuration:
+
+    FLASK_CONF=/srv/app/configs/my-config.py
+
+**!!! NOTICE !!!** - There is a very specific order in which different sources are loaded. Take careful notice to avoid confusion!
+
+- The config file **/srv/app/configs/default.py** is loaded.
+- The **FLASK_CONF** environment variable is loaded if configured.
+- Any instance specific configuration provided via code instantiation is loaded.
+- Settings are loaded individually from any environment variables present.
+- If all required MySQL settings are present in the app.config object and no SQLALCHEMY_DATABASE_URI app setting has been configured, the app.config setting will be automatically created from the provided MySQL settings.
+  - Otherwise, the SQLALCHEMY_DATABASE_URI app setting is set to "sqlite:////srv/app/pdns.db"
+
+### Deploy with Docker Run
+
+To run a simple container on Docker with this image, execute the following Docker command;
+
+    docker run -d -p 8080:80 -e PDAC_SIGNUP_ENABLED=true ngoduykhanh/powerdns-nameserver
+
+### Deploy with Docker Compose
+
+To run this image using Docker Compose, create a YAML file in a place of your choosing and add the following contents to it;
+
+    version: "3.3"
+    services:
+      powerdns-admin:
+        image: ngoduykhanh/powerdns-admin
+        restart: unless-stopped
+        environment:
+          - PDAC_SALT=ntR8BmMQb7MKTJ5YrgQwZr3pxU3uLG
+          - PDAC_SECRET_KEY=eJ92QLuennVStHTj5FVwxz7m4r3Ng3
+        ports:
+          - "8080:80"
+
+Then execute the following Docker Compose command;
+
+    docker-compose -f /path/to/yaml/file.yml up
+
+**!!! NOTICE !!!** - Make sure to set the environment variable `PDAC_SECRET_KEY` to a long random string ([see here for additional information](https://flask.palletsprojects.com/en/1.1.x/config/#SECRET_KEY))
+
+## Building this image
+
+If you want to build this image yourself, you can easily do so using the **docker/bin/build-image** command I have included.
```

Comment:
```suggestion
If you want to build this image yourself, you can easily do so using the **docker/bin/build-image** script included in the repository.
```

Reply:
Great catch, I have updated this mistake. As you can tell, it has been quite some time since I have worked in a team environment. =)